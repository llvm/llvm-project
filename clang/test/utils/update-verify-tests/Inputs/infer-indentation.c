void foo() {
         //     expected-error@+1    2      {{use of undeclared identifier 'a'}}
    a = 2; a = 2; b = 2; b = 2; c = 2;
         //     expected-error@+1    2      {{asdf}}
    d = 2;
    e = 2; f = 2;                 //     expected-error    2      {{use of undeclared identifier 'e'}}
}

