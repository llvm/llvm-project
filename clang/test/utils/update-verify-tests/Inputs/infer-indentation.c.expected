void foo() {
         //     expected-error@+3          {{use of undeclared identifier 'c'}}
         //     expected-error@+2    2      {{use of undeclared identifier 'b'}}
         //     expected-error@+1    2      {{use of undeclared identifier 'a'}}
    a = 2; a = 2; b = 2; b = 2; c = 2;
         //     expected-error@+1          {{use of undeclared identifier 'd'}}
    d = 2;
    //     expected-error@+1          {{use of undeclared identifier 'f'}}
    e = 2; f = 2;                 //     expected-error          {{use of undeclared identifier 'e'}}
}

